plasmapy/utils/__init__.py

```diff
@@ -2,36 +2,3 @@
 Package of functions and classes used to develop clean, readable, and informative
 code.
 """
```

Comment:
```suggestion
"""
    __all__ = [
        "code_repr",
        "datatype_factory_base",
        "decorators",
        "exceptions",
        "roman",
    ]

from plasmapy.utils import (
    code_repr,
    datatype_factory_base,
    decorators,
    exceptions,
    roman,
)
```
I'm currently inclined to keep these imports here and in `__all__` to help with findability of the modules, and also to make sure everything still shows up in the documentation.  Thank you again!

Reply:
Reverted!